### ISSUE TITLE
Basic documentation of Inline Message

### ISSUE THREAD

**Issue by DanielleRameau** [sl-design-system/components#946]
```[tasklist]
### Tasks
- [x] ux docs
- [x] a11y docs
- [x] code docs
- [ ] https://github.com/sl-design-system/components/issues/610
```

**Comment by anna-lach** [comment]
1. We need to change examples in the skeleton docs - `aria-label` should be added to the wrapper, not to every `sl-skeleton` component and it also needs to be changed in the accessibility tab (aria-label for wrapper/s only).

3. `aria-busy` should be optional, needs to be changed in the component as well, so it should not be added in the `sl-skeleton` by default.
